x/twap/migrate_test.go

```diff
@@ -93,3 +93,189 @@ func (suite *TestSuite) TestTwapRecord_GeometricTwap_MarshalUnmarshal() {
 	suite.Require().False(originalRecord.GeometricTwapAccumulator.IsNil())
 	suite.Require().Equal(sdk.ZeroDec(), originalRecord.GeometricTwapAccumulator)
 }
+
+func (suite *TestSuite) TestInitializeGeometricTwap() {
+	var (
+		expectedGeometricAccumUpdate = sdk.OneDec()
```

Comment:
nit suggestion please feel free to resolve this / disregard this
```suggestion
		defaultGeometricAccumUpdateVal = sdk.OneDec()
```